ref/addons.md

```diff
@@ -62,6 +63,15 @@ The following figure shows when the different hooks are executed in the life cyc
 ![Hooks execution in the life cycle of a container|471x601](https://assets.ubuntu.com/v1/bc9b1291-addons-reference-hook-order.png)
 
 ### Hook timeouts
-All hooks are subject to a 5 minute timeout to avoid blocking a container for too long.
+All hooks are subject to a 5 minute timeout by default to avoid blocking a container for too long. The timeout can be configured with `hooks.timeout` key in the addon manifest. For example:
```

Comment:
```suggestion
By default, all hooks are subject to a 5 minute timeout to avoid blocking a container for too long. The timeout can be configured through the `hooks.timeout` key in the addon manifest. For example:
```